plugins/out_azure/azure_conf.c

```diff
@@ -235,3 +249,18 @@ int flb_azure_conf_destroy(struct flb_azure *ctx)
 
     return 0;
 }
+
+int bool_value(const char *v)
```

Comment:
Could you remove this function ?
We already support `flb_utils_bool` function.

Reply:
Please remove this function to reduce maintenance cost.

Reply:
I'm sorry I thought it was deleted. Must be in staging phase 🙈

Reply:
Thank you.